src/main/java/org/spongepowered/common/entity/player/SpongeUser.java

```diff
@@ -183,79 +184,83 @@ public DataContainer toContainer() {
                 .set(DataQueries.USER_SPAWNS, this.spawnLocations);
     }
 
+    private NotImplementedException createException() {
```

Comment:
Maybe this should be named createOfflinePlayerException or notImplementedException, because currently its quite hard to guess the meaning of this Exception from just its name.